tfx_addons/copy_example_gen/README.md

```diff
@@ -10,31 +10,73 @@
 **Project name:** CopyExampleGen component
```

Comment:
Add copyright to the readme. Use The Tensorflow Authors (not Google -- see other files in addons).

Reply:
I do not see any copyright lines in other readme files. May you link me to an example? @michaelwsherman